topic0/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add the .gitignore file to exclude the classpath and compilated files in /target